frontend/pom.xml

```diff
@@ -61,6 +61,17 @@
       <scope>runtime</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.maps</groupId>
+      <artifactId>google-maps-services</artifactId>
+      <version>0.14.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
```

Comment:
Just realized that you probably set this because of a maps service dependency. In that case, since we're committed to using slf4j anyway (if we want Maps API logging), you should also add the above dependency I mentioned. This unifies all our logging to use the same. (slf4j) backend.